lightning/src/events/mod.rs

```diff
@@ -1855,6 +1865,8 @@ impl MaybeReadable for Event {
 					let mut onion_fields = None;
```

Comment:
Maybe it's already obvious but for clarity we could suffix the old fields with `_legacy`.

Reply:
Yep, that sounds cleaner! Done [pr3655.03](https://github.com/shaavan/rust-lightning/commits/pr3655.03)
Thanks! ✨